pkgs/development/python-modules/pwntools/default.nix

```diff
@@ -14,11 +14,14 @@ buildPythonPackage rec {
     sha256 = "609b3f0ba47c975f4dbedd3da2af4c5ca1b3a2aa13fb99240531b6a68edb87be";
   };
 
-  propagatedBuildInputs = [ Mako packaging pysocks pygments ROPGadget capstone paramiko pip psutil pyelftools pypandoc pyserial dateutil requests tox pandoc unicorn intervaltree ];
+  propagatedBuildInputs = [ Mako packaging pysocks pygments ROPGadget capstone paramiko pip psutil pyelftools pyserial dateutil requests tox unicorn intervaltree ];
 
   disabled = isPy3k;
   doCheck = false; # no setuptools tests for the package
 
+  # Can be removed when 3.12.0 is released
+  patches = [ ./remove-pypandoc.patch ];
```

Comment:
Better use `fetchPatch` with https://github.com/Gallopsled/pwntools/pull/1098.patch.

Reply:
Thanks, I didn't realize it existed.